fix: shrink memory lane, glitchy scrub, bad intersection

midzelis opened the pull request:
* Set scrub touch area to 20 from right of screen px on mobile web. 
* Shrink memory-lane on mobile web
* Fix topWindow calculation for intersecting assets
* Fix jumpy onScrubStop behavior.

midzelis committed:
fix: shrink memory lane, glitchy scrub, bad intersection

midzelis committed:
Merge remote-tracking branch 'origin/main' into misc_tweaks

midzelis committed:
readd scrollcomp

midzelis committed:
Fix jitter

midzelis committed:
Merge branch 'main' into misc_tweaks

midzelis committed:
Merge remote-tracking branch 'origin/main' into misc_tweaks

midzelis committed:
prefix duplicate key on upload asset to not existing bucket

midzelis committed:
wip

midzelis committed:
Merge remote-tracking branch 'origin/main' into misc_tweaks

midzelis committed:
Merge remote-tracking branch 'origin/main' into misc_tweaks

alextran1502 committed:
Merge branch 'main' into misc_tweaks

abhinav96 committed:
feat(server): Avoid face match with people born after file creation #4743 (#16918)

* feat(server): Avoid face matching with people born after file creation date (#4743)

* lint

* add medium tests for facial recognition

---------

Co-authored-by: Alex <alex.tran1502@gmail.com>

alextran1502 commented:
Just tested this PR. There is some issue with the skeleton bucket rendered way below the current viewport with the gap in between. This causes a layout shift

<img width="462" alt="image" src="https://github.com/user-attachments/assets/0a90fdf1-751d-4332-9930-f4b159f51430" />

zackpollard committed:
fix: metadata service init failure should halt server startup (#17356)

alextran1502 committed:
chore: post release tasks (#17341)

mertalev committed:
fix(web): use original image if web compatible (#17347)

* use original image if web compatible

* add e2e

* fix shared link handling

* handle redirect in e2e

* fix size not being passed to thumbnail url

* test fullsize in e2e

shenlong-tanwen committed:
refactor(mobile): device asset entity to use modified time (#17064)

* refactor: device asset entity to use modified time

* chore: cleanup

* refactor: remove album media dependency from hashservice

* refactor: return updated copy of asset

* add hash service tests

* chore: rename hash batch constants

* chore: log the number of assets processed during migration

* chore: more logs

* refactor: use lookup and more tests

* use sort approach

* refactor hash service to use for loop instead

* refactor: rename to getByIds

---------

Co-authored-by: shenlong-tanwen <139912620+shalong-tanwen@users.noreply.github.com>

zackpollard committed:
fix: missing index and geodata import process uses normal table (#17343)

* chore: add geodata indexes to table definitions

* chore: rename incorrectly name geodata index

* fix: import into geodata places with correct index names

bo0tzz committed:
chore: remove demo box spec from README.md (#17367)

shenlong-tanwen committed:
fix(mobile): bump isar maxSize (#17372)

shenlong-tanwen committed:
fix(mobile): ignore invalid store keys (#17370)

Co-authored-by: shenlong-tanwen <139912620+shalong-tanwen@users.noreply.github.com>

martabal committed:
fix: improve initial loading time (#17379)

mertalev committed:
chore: use valkey (#17396)

use valkey

danieldietzler committed:
fix: reset memories on logout (#17405)

mmomjian committed:
fix(deployment): warning for database on network share (#17412)

Update example.env

l0ll098 committed:
docs: fixed a wrong path in CLI docs page (#17369)

docs: fixed a wrong path in CLI page

Zlendy committed:
fix(web): Video memories are played at 100% volume instead of respecting user preference (#17424)

shenlong-tanwen committed:
fix(mobile): prevent unnecessary reload on multi user timeline (#17418)

Co-authored-by: shenlong-tanwen <139912620+shalong-tanwen@users.noreply.github.com>

benmccann committed:
chore(web): switch to writable derived one more place (#17399)

weblate committed:
chore(web): update translations (#17254)

Translate-URL: https://hosted.weblate.org/projects/immich/immich/bg/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/bi/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/ca/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/cs/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/de/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/el/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/es/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/et/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/fa/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/fi/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/fr/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/gl/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/he/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/hr/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/hu/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/it/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/ja/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/ka/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/lv/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/ms/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/nl/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/pl/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/pt/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/ro/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/ru/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/sl/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/sr_Cyrl/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/sr_Latn/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/sv/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/ta/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/te/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/th/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/tr/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/uk/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/vi/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/immich/immich/zh_SIMPLIFIED/
Translation: Immich/immich

Co-authored-by: Bonov <bonov@mail.ru>
Co-authored-by: C D <chinnidiwakar5@gmail.com>
Co-authored-by: Daniel Correa Lobato <daniel@lobato.org>
Co-authored-by: Emre Saraçoğlu <hello@emresaracoglu.com>
Co-authored-by: Fjuro <fjuro@users.noreply.hosted.weblate.org>
Co-authored-by: GND <jehende@jehende.fr>
Co-authored-by: Gocha Gulua <gocha.gulua@gmail.com>
Co-authored-by: Hurricane-32 <rodrigorimo@hotmail.com>
Co-authored-by: Indrek Haav <indrek.haav@hotmail.com>
Co-authored-by: Leigh van der merwe <palitu822@gmail.com>
Co-authored-by: LennartWeinzierl <lennart.weinzierl@gmx.de>
Co-authored-by: Leo Bottaro <github@leobottaro.com>
Co-authored-by: Luis Peregrina <luis.a.peregrina@gmail.com>
Co-authored-by: Matjaž T <matjaz@moj-svet.si>
Co-authored-by: Miki Mrvos <medolino2009@gmail.com>
Co-authored-by: Mārtiņš Bruņenieks <martinsb@gmail.com>
Co-authored-by: Oleksandr Zhukov <aleksandr.a.zhukov@gmail.com>
Co-authored-by: Passawish Paktiwong <passawishp@outlook.com>
Co-authored-by: Petri Hämäläinen <petri.hamalainen@mailbox.org>
Co-authored-by: Ruben Hensen <ruben.hensen@protonmail.com>
Co-authored-by: Runskrift <anders@rimfrost.nu>
Co-authored-by: Shawn <xiaxinx@gmail.com>
Co-authored-by: Stein-Aksel Basma <stein-aksel@basma.no>
Co-authored-by: Sylvain Pichon <service@spichon.fr>
Co-authored-by: Tachibana Saza <tachibanasaza@proton.me>
Co-authored-by: Temuri Doghonadze <temuri.doghonadze@gmail.com>
Co-authored-by: Theofilos Nikolaou <th.nikolaou@gmail.com>
Co-authored-by: User 123456789 <w0g-1es-5qq@cld3.com>
Co-authored-by: Vin <k3kelm4vw@mozmail.com>
Co-authored-by: aks-cadesign <aks@cadesignbase.dk>
Co-authored-by: eav5jhl0 <eav5jhl0@users.noreply.hosted.weblate.org>
Co-authored-by: grgergo <gergo_g@proton.me>
Co-authored-by: late <late@users.noreply.hosted.weblate.org>
Co-authored-by: millallo <millallo@tiscali.it>
Co-authored-by: przmkg <przemek@gasinski.eu>
Co-authored-by: thehijacker <thehijacker@gmail.com>
Co-authored-by: timmy61109 <qazzxcasdqwewsxedc@gmail.com>
Co-authored-by: waclaw66 <waclaw66@seznam.cz>
Co-authored-by: xuars <yago.rana.gayoso@gmail.com>
Co-authored-by: Вячеслав Лукьяненко <madeinchuguev@gmail.com>

rubenhensen committed:
feat: allow accounts with a quota of 0 GiB (#17413)

* Allow 0GiB quotas in user create/edit form, remove unused translations

* Make requireQuota check for null or 0

* Add unlimited quota change to the docs

* Fix user dto formatting

* Fix formating edit-user-form

* Regenerate open-api files

* Revert unnecessary i18n file changes

* Re-add newline en.json

* Resolve linting issues

* Fix formatting edit-user-form

* Re-add manifest

Sese-Schneider committed:
fix(mobile): video player restarting when device rotates (#17362)

* fix(mobile): Video player restarting when device rotates

* use global key in state

* Implement suggestions from code review

YarosMallorca committed:
fix(mobile): items not deselecting on back button (#17403)

* fix: items not deselecting on back button

* chore: add comments

shenlong-tanwen committed:
fix(mobile): use custom filter to fetch asset path entities (#17344)

Co-authored-by: shenlong-tanwen <139912620+shalong-tanwen@users.noreply.github.com>

shenlong-tanwen committed:
fix(mobile): exifInfo not updated on sync (#17407)

* fix(mobile): exifInfo not updated on sync

* add tests

---------

Co-authored-by: shenlong-tanwen <139912620+shalong-tanwen@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update dependency vite to v6.2.5 [security] (#17391)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

jrasm91 committed:
feat: extension, triggers, functions, comments, parameters management in sql-tools (#17269)

feat: sql-tools extension, triggers, functions, comments, parameters

bo0tzz committed:
fix: restore mangled footnotes (#17446)

I broke this in #17257

jrasm91 committed:
refactor: remove natural earth countries enity (#17445)

jrasm91 committed:
refactor: stop using geodata entity type (#17444)

mPyKen committed:
fix(server): Exclude album assets in shared link payload (#17207)

* fix(server): Exclude album assets in shared link payload

* Fix e2e test

renovate[bot] committed:
chore(deps): update dependency @types/node to ^22.14.0 (#17459)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update grafana/grafana docker tag to v11.6.0 (#17460)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update github/codeql-action digest to 45775bd (#17452)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update mcr.microsoft.com/devcontainers/typescript-node:22 docker digest to b0b88ef (#17453)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

devsnek committed:
feat(web): improve individual share ux (#17430)

midzelis committed:
Fix disappearing bucket, longpress, dynamic viewport sizes

midzelis committed:
Merge remote-tracking branch 'origin/main' into misc_tweaks

midzelis committed:
bad merge

midzelis committed:
bug

midzelis committed:
overscroll

midzelis committed:
Keyboard Accessible Scrubber

midzelis committed:
Merge remote-tracking branch 'origin/main' into misc_tweaks

midzelis commented:
Closing this - it has been obsoleted by other, smaller, PRs
